io.gitlab.celleron56.libretile.yaml

```diff
@@ -0,0 +1,92 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/files/start.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+
+modules:
+  - name: libretile
+    buildsystem: simple
+    
+    build-commands:
+       - mkdir /app/files
+       - mv * /app/files
+       - echo "python3 /app/files/main.py" >> /app/files/start.sh 
+       - chmod 777 /app/files/start.sh
+       - mkdir /app/share
+       - mkdir /app/share/metainfo
+       - mkdir -p /app/share/icons/hicolor/64x64/apps
+       - mkdir -p /app/share/icons/hicolor/128x128/apps
+       - mkdir /app/share/applications/
+       - mv /app/files/util/io.gitlab.celleron56.libretile.metainfo.xml /app/share/metainfo/io.gitlab.celleron56.libretile.metainfo.xml
+       - mv /app/files/util/libretile.png /app/share/icons/hicolor/64x64/apps/io.gitlab.celleron56.libretile.png
+       - mv /app/files/util/libretile128.png /app/share/icons/hicolor/128x128/apps/io.gitlab.celleron56.libretile.png
+       - mv /app/files/util/io.gitlab.celleron56.libretile.desktop  /app/share/applications/io.gitlab.celleron56.libretile.desktop
```

Comment:
the exec line in that .desktop file is invalid. I should just be the name of the executable.

Reply:
so the start.sh file ? (it is needed because else the build wuold say i need a shell wrapper )

Reply:
also why do you have two desktop file?

Reply:
it should be only 1 idk why there are 2 

i'll remove whichever one is the duplicate